src/zeit/retresco/connection.py

```diff
@@ -102,6 +105,15 @@ def get_topicpage_documents(self, id, start=0, rows=25):
             result.append(page)
         return result
 
+    def get_article_body(self, uuid, timeout=None):
+        __traceback_info__ = (uuid,)
+        try:
+            response = self._request(
+                'GET /content/{}'.format(urllib.quote(uuid)), timeout=timeout)
```

Comment:
Bist Du sicher, dass das der richtige Endpoint ist? Im Zeit-TMS-Konzept-1.1.pdf ist nämlich von `GET /api/in-text-linked-documents/<id>` bzw. sogar direkt `GET /api/in-text-linked-documents/<id>/body.html` die Rede.

.oO(frag mich aber nicht, an welcher kanonischen Stelle Du diese Doku hättest finden können... seufz)